packages/gcr.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Gcr < Package
+  description 'GNOME crypto package'
+  homepage 'https://www.gnome.org'
+  version '3.36.0'
+  compatibility 'all'
+  source_url 'https://ftp.gnome.org/pub/GNOME/sources/gcr/3.36/gcr-3.36.0.tar.xz'
+  source_sha256 'aaf9bed017a2263c6145c89a1a84178f9f40f238426463e4ae486694ef5f6601'
+
+  binary_url ({
+     armv7l: 'https://www.dropbox.com/s/ggpe4y6txpveomi/gcr-3.36.0-chromeos-armv7l.tar.xz',
+  })
+  binary_sha256 ({
+     armv7l: '84a9cb611f086c42cc920b2937ca8cb1c2287ae818ed58f61631bef421e6b41f',
+  })
```

Comment:
You don't need to include the binary but nice to know you tested the build. :+1:

Reply:
Understood, I'll remove it!